### DISCUSSION TITLE
winzip install task schedule errors

### DISCUSSION THREAD

**Discussion by anmac1789** [sandboxie-plus/Sandboxie#4327] [General]
sandboxie plus crashes when trying to install latest version of winzip due to task schedule error. Please see screenshot. How do I fix this ? it appears to me somekind of permissions issue

![image](https://github.com/user-attachments/assets/e752e702-e862-4e9d-88c3-dd62a48c89ec)

**Comment by isaak654** [comment]
Duplicate of #4328
